### DISCUSSION TITLE
StarWar girl, May the dark side of the force be with her ^^, share Prompt

### DISCUSSION THREAD

**Discussion by kaol2046** [fiatrete/SDCN-Stable-Diffusion-Computing-Network#53] [Show and tell]
![02162-371951813-(Darth Vader_1 18) complex 3d render ultra detailed of a beautiful porcelain profile woman android face, cyborg, robotic parts,](https://user-images.githubusercontent.com/2410035/232190672-359ea13e-5ecf-4c4a-a21e-e82deb1a23f4.png)

SD Prompt:
(Darth Vader:1.18) complex 3d render ultra detailed of a beautiful porcelain profile woman android face, cyborg, robotic parts, 150 mm, beautiful studio soft light, rim light, vibrant details, luxurious cyberpunk, lace, anatomical, facial muscles, cable electric wires, microchip, elegant, beautiful background, octane render, 8k, best quality, masterpiece, illustration, an extremely delicate and beautiful, extremely detailed ,CG ,unity ,wallpaper, (realistic, photo-realistic:1.37),Amazing, finely detail, masterpiece,best quality,official art, extremely detailed CG unity 8k wallpaper, absurdres, incredibly absurdres, robot , stading, full body (SFW) <lora:koreanDollLikeness_v10:0.5>
Negative prompt: sketches, (worst quality:2), (low quality:2), (normal quality:2), lowres, normal quality, ((monochrome)), ((grayscale)), skin spots, acnes, skin blemishes, bad anatomy,(fat:1.2),facing away, looking away,tilted head, {Multiple people}, lowres,bad anatomy,bad hands, text, error, missing fingers,extra digit, fewer digits, cropped, worstquality, low quality, normal quality,jpegartifacts,signature, watermark, username,blurry,bad feet,cropped,(poorly drawn hands),poorly drawn face,mutation,deformed,worst quality,low quality,normal quality,jpeg artifacts,signature,watermark, (extra fingers) ,(fewer digits) ,extra limbs,extra arms,extra legs,malformed limbs,fused fingers,too many fingers,long neck,cross-eyed,(mutated hands),polar lowres,bad body,bad proportions,gross proportions,text,error,missing fingers,missing arms,missing legs,extra digit, extra arms, extra leg, extra foot
Steps: 33, Sampler: DPM++ 2M Karras, CFG scale: 8, Seed: 371951813, Size: 512x784, Model hash: fc2511737a, Denoising strength: 0.5, Clip skip: 2, ENSD: 31337, Hires upscale: 1.5, Hires steps: 15, Hires upscaler: Latent (bicubic antialiased)

welcome to:  http://mygirls.world

**Comment by voltff** [comment]
like the legs...
